### PR TITLE
Update node xml

### PR DESCRIPTION
**Pull request by madhuBayy**
@ldko @somexpert 
This PR adds coda's `Node` model's `status` to the node XML in feed as per [this](https://github.com/unt-libraries/coda/pull/152#pullrequestreview-495845460).

### CODE REVIEW

**Comment by madhuBayy** [comment]
This PR needs some modifications. Please hold off your review until then.

**Comment by madhuBayy** [comment]
@somexpert @ldko This is ready for review now.

**Comment by ldko** [comment]
I think the code looks fine. Would you add a line in the Changelog about "Add support for Node status attribute." or something like that?

**Review by ldko — APPROVED** [review]
Looks good to me! Thanks, Madhulika! :+1:

File: codalib/bagatom.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def nodeToXML(nodeObject):
     Take a Django node object from our CODA store and make an XML
     representation
     """
-
+    nodeObject_status = {'0': 'Inactive', '1': 'Active'}
```
Comment (somexpert) [inline codalib/bagatom.py:205]: I'd prefer we didn't hardcode this dictionary here, as then we have two places to change (in two different repos) if the original choices undergo some alteration. Instead, if we're just after the status wording, why not just use `nodeObject.get_status_display()` on line 222 as we did in the templates in coda?

**Review by somexpert — COMMENTED** [review]
Just the one comment from me

File: codalib/bagatom.py (hunk 1 above)
Comment (madhuBayy) [reply codalib/bagatom.py:205]: I don't think we can use it here because codalib is not a django project.

Comment (somexpert) [reply codalib/bagatom.py:205]: codalib isn't a Django project, but the docstring indicates that this function is expecting to be called _from_ a Django project, so I think we could still use this, but I certainly could be wrong. Any thoughts on this, @ldko?

Comment (ldko) [reply codalib/bagatom.py:205]: For questions like this, I would try it out:
```
>           statusNode.text = nodeObject.get_status_display()
E           AttributeError: 'NodeStub' object has no attribute 'get_status_display'
```
and if I look at the members of `nodeObject` in the nodeeToXML I don't see that method:
```
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', 'last_checked', 'node_capacity', 'node_name', 'node_path', 'node_size', 'node_url', 'status']
```
So, I think Madhulika might be right.

Comment (somexpert) [reply codalib/bagatom.py:205]: Okie dokie, I stand corrected. I withdraw this comment :smile:

**Review by somexpert — APPROVED** [review]
LGTM :thumbsup:

**Pull request merged**